jdi-light/src/main/java/com/epam/jdi/light/common/JDIAction2.java

```diff
@@ -0,0 +1,33 @@
+package com.epam.jdi.light.common;
+
+/**
+ * Created by Roman Iovlev on 14.02.2018 Email: roman.iovlev.jdi@gmail.com; Skype: roman.iovlev
+ */
+
+import static com.epam.jdi.light.logger.LogLevels.STEP;
+
+import com.epam.jdi.light.logger.LogLevels;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+
+/**
+ * Created by Roman Iovlev on 26.09.2019
+ * Email: roman.iovlev.jdi@gmail.com; Skype: roman.iovlev
```

Comment:
remove JDIAction2 (it was need for research part)

Reply:
Done